oaknational/Oak-Web-Application
# Style/header

## Description

- changes header search form to use `Input` (as in the designs)
- adds optional icon for input

## Issue(s)

Re #183

## How to test

1. Go to https://samara-storybook-gbfp7fibf-oak-national-academy.vercel.app/?path=/story/form-input--with-icon
4. You should see input with icon like in screenshots below
5. When typing into the input, both the border and icon should change to their respective "active" colours 

## Screenshots

### Before/after
![image](https://user-images.githubusercontent.com/12934669/171143013-348f270f-4b2b-4991-8206-26988f1e22a6.png)


![image](https://user-images.githubusercontent.com/12934669/170759669-e0b6de49-076e-4663-9569-0ddaefa38083.png)
![image](https://user-images.githubusercontent.com/12934669/170759802-9a312952-f1af-4655-8582-4e7d6447d137.png)


## Checklist
